src/components/screens/ScoreSetEditor.vue

```diff
@@ -1282,7 +1284,27 @@ export default {
         this.secondaryPublicationIdentifiers = []
         this.publicationIdentifiers = []
         this.dataUsagePolicy = null
-        this.targetGene = null
+        this.existingTargetGene = null
+        this.targetGene = {
```

Comment:
this is redundant with l686 where target gene is created. I think you want to add a helper function emptyTargetGene outside the component to wrap this code and then find all the similar code and replace it with this.

Reply:
Good idea.